test/validate/eia_test.py

```diff
@@ -37,10 +37,9 @@
         # EIA860 tables
         "core_eia860__assn_boiler_generator",
         "out_eia__yearly_boilers",
-        "_out_eia__yearly_generators",
+        "out_eia__yearly_generators",
```

Comment:
Huh, this wasn't here before? Glad you caught that.

Reply:
Oh there's a bajillion things that aren't here.